app/models/product.rb

```diff
@@ -31,6 +31,10 @@ def payload_for(data, type:)
           "properties" => data["attributes"],
           "product_category_id" => category_id,
         }
+      when "GoogleProduct"
+        {
+          "product_category_id" => data["product_category_id"],
+        }
```

Comment:
I don't think we need this given the default case is `data.slice("product_category_id")`

```suggestion
```

Reply:
Removed.